{
  "debianbug": 358892,
  "description": "The attachment scrubber (Scrubber.py) in Mailman 2.1.5 and earlier, when using Python's library email module 2.5, allows remote attackers to cause a denial of service (mailing list delivery failure) via a multipart MIME message with a single part that has two blank lines between the first boundary and the end boundary.",
  "releases": {
    "bullseye": {
      "fixed_version": "2.1.6-1",
      "repositories": {
        "bullseye": "1:2.1.29-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "2.1.6-1",
      "repositories": {
        "buster": "1:2.1.29-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "2.1.6-1",
      "repositories": {
        "jessie": "1:2.1.18-2+deb8u2",
        "jessie-security": "1:2.1.18-2+deb8u4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "2.1.6-1",
      "repositories": {
        "sid": "1:2.1.29-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "2.1.6-1",
      "repositories": {
        "stretch": "1:2.1.23-1+deb9u4",
        "stretch-security": "1:2.1.23-1+deb9u3"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}